tools/check-cyclic-dependencies.js

```diff
@@ -1,57 +1,72 @@
+    "frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view_helpers.ts",
+    "frontend/javascripts/oxalis/model/accessors/skeletontracing_accessor.ts",
   ],
+  ["frontend/javascripts/libs/request.ts", "frontend/javascripts/admin/datastore_health_check.ts"],
   [
-    'frontend/javascripts/libs/request.ts',
-    'frontend/javascripts/admin/datastore_health_check.ts'
+    "frontend/javascripts/admin/admin_rest_api.ts",
+    "frontend/javascripts/libs/request.ts",
+    "frontend/javascripts/admin/datastore_health_check.ts",
   ],
   [
-    'frontend/javascripts/admin/admin_rest_api.ts',
-    'frontend/javascripts/libs/request.ts',
-    'frontend/javascripts/admin/datastore_health_check.ts'
+    "frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx",
+    "frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx",
   ],
   [
-    'frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx',
-    'frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx'
+    "frontend/javascripts/admin/organization/upgrade_plan_modal.tsx",
+    "frontend/javascripts/admin/organization/organization_cards.tsx",
   ],
   [
-    'frontend/javascripts/admin/organization/upgrade_plan_modal.tsx',
-    'frontend/javascripts/admin/organization/organization_cards.tsx'
+    "frontend/javascripts/admin/task/task_create_form_view.tsx",
+    "frontend/javascripts/admin/task/task_create_bulk_view.tsx",
   ],
   [
-    'frontend/javascripts/admin/task/task_create_form_view.tsx',
-    'frontend/javascripts/admin/task/task_create_bulk_view.tsx'
+    "frontend/javascripts/admin/team/team_list_view.tsx",
+    "frontend/javascripts/admin/team/edit_team_modal_view.tsx",
   ],
   [
-    'frontend/javascripts/admin/team/team_list_view.tsx',
-    'frontend/javascripts/admin/team/edit_team_modal_view.tsx'
+    "frontend/javascripts/dashboard/advanced_dataset/dataset_table.tsx",
+    "frontend/javascripts/dashboard/folders/folder_tree.tsx",
   ],
   [
-    'frontend/javascripts/dashboard/advanced_dataset/dataset_table.tsx',
-    'frontend/javascripts/dashboard/folders/folder_tree.tsx'
+    "frontend/javascripts/oxalis/model_initialization.ts",
+    "frontend/javascripts/oxalis/controller/url_manager.ts",
   ],
   [
-    'frontend/javascripts/oxalis/model_initialization.ts',
-    'frontend/javascripts/oxalis/controller/url_manager.ts'
+    "frontend/javascripts/oxalis/geometries/plane.ts",
+    "frontend/javascripts/oxalis/geometries/materials/plane_material_factory.ts",
+    "frontend/javascripts/oxalis/shaders/main_data_shaders.glsl.ts",
   ],
   [
-    'frontend/javascripts/oxalis/geometries/plane.ts',
-    'frontend/javascripts/oxalis/geometries/materials/plane_material_factory.ts',
-    'frontend/javascripts/oxalis/shaders/main_data_shaders.glsl.ts'
-  ]
+    "frontend/javascripts/libs/toast.tsx",
+    "frontend/javascripts/libs/render_independently.tsx ",
+    "frontend/javascripts/oxalis/throttled_store.ts ",
```

Comment:
where does render_independently import throttled store? I don't see this in the code :thinking:

Reply:
Yeah it also took me a few seconds to find :eyes:.

The import is done in-time when needed in line 14: https://github.com/scalableminds/webknossos/blob/c47cd6500e653e9b36ec102f6b7bfee9500a45ad/frontend/javascripts/libs/render_independently.tsx#L9-L22